scripts/lib/CIME/utils.py

```diff
@@ -199,8 +199,7 @@ def _convert_to_fd(filearg, from_dir):
 _hack=object()
 
 def run_sub_or_cmd(cmd, cmdargs, subname, subargs, logfile=None, case=None,
-                   input_str=None, from_dir=None, verbose=None,
-                   arg_stdout=_hack, arg_stderr=_hack, env=None, combine_output=False):
+                   from_dir=None, combine_output=False):
```

Comment:
Do you think a user will ever not want to see errput from buildnml? If not, than we should removed the arg from this function and just set it to True in the run_cmd_no_fail call below.

Reply:
buildnml isn't necessarily the only user of run_sub_or_cmd

Reply:
Oh ok.